egs/cfg2/wip/expand_tree_test.py

```diff
@@ -1,4 +1,6 @@
 #! /usr/bin/env ipython
 from kaldi_grammar import KaldiGrammar
 k = KaldiGrammar('current_grammar.fcfg','T')
-words = k.expand_tree()
\ No newline at end of file
+# words = k.expand_tree()
+recognised_word = 'say'
+filter = k.check_word(recognised_word)
```

Comment:
Don't use filter as variable name as it is an internal function of python.

Reply:
This script is only for testing. After the last commit the file will be deleted (but good remark, anyway) :)